packages/ui/src/mixins/SizeMixin.ts

```diff
@@ -42,7 +42,7 @@ class SizeProps {
 export class SizeMixin extends mixins(Vue.with(SizeProps)) {
   fontRegex = /(?<fontSize>\d+)(?<extension>px|rem)/i
 
-  get sizesConfigGlobal () {
+  get sizesConfigGlobal (): any {
```

Comment:
We can make types for SizesConfig if we don't have them already. 

SizesConfig | undefined